src/cljs/webchange/common/image_selector/form/views.cljs

```diff
@@ -7,17 +7,20 @@
 
 (defn- tag-selector
   [{:keys [id]}]
-  (let [options @(re-frame/subscribe [::state/tags-options id])
-        handle-change (fn [{:keys [value checked?]}]
+  (let [options (into [] @(re-frame/subscribe [::state/tags-options id]))
```

Comment:
What is the meaning of `into` here? Probably it should be `(vec)` inside `tags-options` sub..